Gemfile

```diff
@@ -34,6 +34,9 @@ gem 'rack-test', '~> 0.6.3', require: 'rack/test'
 gem 'clinic_finder', '~> 0.0.1'
 gem 'geokit'
 gem 'secure_headers', '~> 3.6', '>= 3.6.4'
+gem 'kaminari-mongoid'
+gem 'kaminari'
```

Comment:
Haven't heard of this lib but it looks sweet! 👍

Reply:
I think ```will_paginate``` didn't support ```mongoid```, this looks cooler and provides cool pagination links.